include/kodlab_mjbots_sdk/common_header.h

```diff
@@ -87,19 +93,28 @@ class ValidatedCache
    * @brief Sets the class data and marks it valid
    * @param data valid data
    */
-  void set(const T &data)
+  virtual void set(const T &data)
   {
     data_ = data;
     valid_ = true;
   }
 
+  /**
+   * @brief Uses operator= to set the class data and marks it valid
+   * @param data valid data
+   */
+  void operator=(const T &data)
+  {
+    set(data);
+  }
+
```

Comment:
I feel like this is fine despite the possible user confusion since the intended usage is to validate on new data and invalidate at a later point.